airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml

```diff
@@ -54,6 +52,33 @@ acceptance_tests:
           board_pins:
             - name: "links"
               bypass_reason: "because it contains non-secure http link, which leads to failed QA tests"
+            - name: "link"
+              bypass_reason: "because it contains non-secure http link, which leads to failed QA tests"
+            - name: "media"
+              bypass_reason: "because it contains non-secure http link, which leads to failed QA tests"
+          board_section_pins:
+            - name: "links"
+              bypass_reason: "because it contains non-secure http link, which leads to failed QA tests"
+            - name: "link"
+              bypass_reason: "because it contains non-secure http link, which leads to failed QA tests"
+            - name: "media"
+              bypass_reason: "because it contains non-secure http link, which leads to failed QA tests"
```

Comment:
We should rather make the QA check tolerate http links in expected records files.

Reply:
@midavadim please review https://github.com/airbytehq/airbyte/pull/32499

Reply:
@alafanechere it would be great to have that PR merged!
Thank you!

Reply:
It is merged, please remove these bypass 🙏 
I think you also have to increase the unit test coverage to get a 🟢 CI